serverless/functions/troubleshooting/handler-not-found.mdx

```diff
@@ -0,0 +1,44 @@
+---
+meta:
+  title: Unable to find the function handler
+  description: Troubleshoot problems with finding the handler when using Scaleway Serverless Functions.
+content:
+  h1: Unable to find the function handler
+  paragraph: Troubleshoot problems with finding the handler when using Scaleway Serverless Functions.
+tags: serverless functions troubleshooting issue error function handler execution run cannot find
+dates:
+  validation: 2024-11-06
+  posted: 2024-11-06
+categories:
+  - serverless
+---
+
+## Problem
+
+My serverless Function is unable to find the handler during its execution, and displays an error message similar to the following:
+
+```
+Function Handler does not exist, check that you provided the right HANDLER parameter …
+```
+
+Or
+
+```
+Provided Handler does not exist, or does not export methods properly
+```
+
+## Cause
+
+The systems cannot find the function that should handle the request. It can be related to an error in your code before your `handle` function.
```

Comment:
```suggestion
The systems cannot find the function that should handle the request. It may be related to an error in your code before your `handle` function.
```